demo/components/demo/components/datatables/API.md

```diff
@@ -3,22 +3,27 @@
 ### Input
 
   * `data: any[]`: Data to be displayed as rows in the table.
-  * `dataTrackByKey?: string`: Unique object property of data, useful for internal tracking.
+  * `sort: Object`: Sorting state object as `{ key: '<column key>', sort: '<asc|desc>'}`
```

Comment:
I think that `dir` or `direction` would be more suitable than `sort` for the `sort` input property.